src/EasyCaching.HybridCache/HybridCachingProvider.cs

```diff
@@ -273,11 +284,13 @@ public CacheValue<T> Get<T>(string cacheKey)
                
                 _localCache.Set(cacheKey, cacheValue.Value, ts);
 
+                TrackCacheStats(cacheKey, cacheValue);
                 return cacheValue;
             }
 
             _logger?.LogDebug("distributed cache can not get the value of {0}", cacheKey);
 
+            TrackCacheStats(cacheKey, CacheValue<T>.NoValue);
```

Comment:
```suggestion
            OnCacheMiss(cacheKey);
```
Так понятнее. TrackCacheStats нужен только когда не не хочется if'чик писать. А тут и в других местах он уже есть.